### ISSUE TITLE
Ignores settings after some time

### ISSUE THREAD

**Issue by dhaunac** [quodroc/HumbleNewTabPage#47]
In the settings tab I disabled everything but "Bookmarks bar" because I just wanted to have the bookmarks there. So "Other bookmarks" and "Other device" is unchecked.

But after some time using (I'm using this app for over a year) those things are still rendered into my menu. This I consider a bug. I can give you my settings and also show a screenshot of my settings opened and the menus displayed.

If you need more help fixing it, I can assist you. Please answer me so I can know if this bug will be fixed eventually or I need to find a new extention for this, because this bug is totally annoying to me.

**Comment by dhaunac** [comment]
I forgot to mention, that if I check and uncheck "Other bookmarks" I will not be displayed anymore. But this is just a temporary fix. Will be displayed after a couple of hours or days.

**Comment by dhaunac** [comment]
{"column.0.0":"1451","column.1.0":"1","open.1":"true","open.893":"true","options.auto_close":"1","options.fade":"0.999","options.font_size":"20","options.hide_options":"0","options.lock":"0","options.number_closed":"5","options.number_top":"10","options.show_1":"1","options.show_2":"0","options.show_3":"0","options.show_apps":"0","options.show_closed":"0","options.show_devices":"0","options.show_recent":"0","options.show_root":"0","options.show_top":"0","options.show_weather":"0","options.theme":"Warm","options.weather_location":"Bonn","options.weather_location_id":"640161"}

**Comment by quodroc** [comment]
Strange. The only thing I can think is if the ID of the folder changes somehow. Do you sync your bookmarks with another device?

**Comment by dhaunac** [comment]
Yes, I use chrome on my desktop and on my mobile phone. Since I'm logged in on both, they sync bookmarks.

**Comment by insintesi** [comment]
This also happens to me with the latest Firefox 57 Beta. Sync is disabled. My custom settings are all lost after a minute or two. Other Firefox add-ons keep their settings.

**Comment by dhaunac** [comment]
For me it helped to delete the extention and set up everything from scratch.
But I would classify it as bug now, since it is not my problem alone anymore.
